resources/definitions/flsun_qq.def.json

```diff
@@ -0,0 +1,48 @@
+{
+	"version": 2,
+	"name": "FLSUN QQ",
+	"inherits": "fdmprinter",
+	"metadata": {
+		"visible": true,
+		"manufacturer": "FLSUN",
+		"author": "Daniel Green",
+		"file_formats": "text/x-gcode",
+		"icon": "icon_ultimaker2",
+		"platform_offset": [0, 0, 0],
+		"machine_extruder_trains": {
+			"0": "flsun_qq_extruder"
+		}
+	},
+	"overrides": {
+		"machine_extruder_count": {
+			"default_value": 1
+		},
+		"machine_heated_bed": {
+			"default_value": true
+		},
+		"machine_width": {
+			"default_value": 240
+		},
+		"machine_height": {
+			"default_value": 285
+		},
+		"machine_depth": {
+			"default_value": 240
+		},
```

Comment:
[This page](https://all3dp.com/1/flsun-qq-3d-printer-review/) claims that the printer has a printing area of 260x260x370mm. You might want to review this. Or did you experimentally find that the usable print area is smaller?

Reply:
These are the settings recommended by the owners manual.